siwe/siwe.py

```diff
@@ -181,10 +169,6 @@ def prepare_message(self) -> str:
 
         suffix_array = [uri_field, version_field, chain_field, nonce_field]
 
-        if self.issued_at is None:
```

Comment:
the pydantic model does allow `None` values of issued_at so this branch is never touched